solr/core/src/test/org/apache/solr/core/ConfigureRecoveryStrategyTest.java

```diff
@@ -82,14 +82,6 @@ public static class CustomRecoveryStrategy extends RecoveryStrategy {
 
     private String alternativeBaseUrlProp;
 
-    public String getAlternativeBaseUrlProp() {
-      return alternativeBaseUrlProp;
-    }
-
-    public void setAlternativeBaseUrlProp(String alternativeBaseUrlProp) {
-      this.alternativeBaseUrlProp = alternativeBaseUrlProp;
-    }
-
```

Comment:
This is used via reflection: https://github.com/apache/solr/blob/releases/solr/9.0.0/solr/core/src/java/org/apache/solr/cloud/RecoveryStrategy.java#L91